src/js/api/auth/login.js

```diff
@@ -59,8 +58,9 @@ export async function login(profile) {
     const { token, role, email, id, ...filteredProfile } = await response.json();
 
     // The following code handles the log in response. part of this handling is to set the Role entry in our storage solution.
-    // before starting the handling we therefore clear the existing value for the Role entry. It will be set to the correct values depending on the current log in response.
-    new Store('Role', 'null', Boolean(remember !== 'on')).clear();
+    // before starting the handling we therefore clear the existing value for the Role entry.
+    // It will be set to the correct values depending on the current log in response.
+    localStorage.setItem('Role', 'null');
```

Comment:
the 'Role'-entry is only read directly from localStorage. As such we don't need the indirection via the Store-class.